### PR TITLE
Expand OS interaction section

### PR DESCRIPTION
**Pull request by bendudson**
Add to "Interfacing with your OS"
- Examples of running external program using `launch-program` and communicating with it
- Some introduction and links to libraries

### CODE REVIEW

File: os.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ title: Interfacing with your OS
 
 
 The ANSI Common Lisp standard doesn't mention this topic. (Keep in mind that it was written at a time where [Lisp Machines](http://kogs-www.informatik.uni-hamburg.de/~moeller/symbolics-info/symbolics.html) were at their peak. On these boxes Lisp _was_ your operating system!) So almost everything that can be said here depends on your OS and your implementation.
+There are however some widely used libraries, which either come with your Common Lisp implementation, or are easily
```
Comment (pnathan) [inline os.md]: `there are, however, some` <-- grammar

File: os.md [hunk 2]
```diff
@@ -361,6 +373,72 @@ and `:stream` causes a stream to be made available via
...
+
+;; Close input and output streams
+* (uiop:close-streams *shell*)
+* (uiop:process-alive-p *shell*)
+NIL
+~~~
+
+### Input and output from subprocess
+
+If the `input` keyword is set to `:stream` then a stream is created
```
Comment (pnathan) [inline os.md]: ``:stream`:, then` <-- +comma

**Comment by pnathan** [comment]
seems good, modulo some `,`

**Comment by bendudson** [comment]
Thanks and Happy New Year! I'm getting too used to missing out commas except in backquotes. Now with more commas.

**Pull request merged**